### ISSUE TITLE
r-arrow 11.0.0 is built without dataset support

### ISSUE THREAD

**Issue by darabos** [conda-forge/r-arrow-feedstock#62]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

When I call `open_dataset()` with `r-arrow` version 11.0.0 installed from `conda-forge`, it fails with this error:

```
Error: This build of the arrow package does not support Datasets 
```

With `r-arrow` 10.0.1 it's fine.

https://issues.apache.org/jira/browse/ARROW-15072 suggests this is controlled by `LIBARROW_MINIMAL`. ([docs](https://arrow.apache.org/docs/r/reference/install_arrow.html))

It's not even clear to me which package is responsible for this. Downgrading `r-arrow` also downgraded `arrow-cpp`, `lib-arrow`, and `pyarrow` in my environment.

### Installed packages

```shell
It's almost 500 packages. They are installed from https://github.com/lynxkite/lynxkite/blob/main/conda-env.yml
```

### Environment info

```shell
active environment : lk2
    active env location : /home/darabos/miniconda3/envs/lk2
            shell level : 3
       user config file : /home/darabos/.condarc
 populated config files : /home/darabos/.condarc
          conda version : 4.12.0
    conda-build version : not installed
         python version : 3.9.15.final.0
       virtual packages : __cuda=11.6=0
                          __linux=5.10.0=0
                          __glibc=2.31=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/darabos/miniconda3  (writable)
      conda av data dir : /home/darabos/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/darabos/miniconda3/pkgs
                          /home/darabos/.conda/pkgs
       envs directories : /home/darabos/miniconda3/envs
                          /home/darabos/.conda/envs
               platform : linux-64
             user-agent : conda/4.12.0 requests/2.27.1 CPython/3.9.15 Linux/5.10.0-21-amd64 debian/11 glibc/2.31
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

**Comment by h-vetinari** [comment]
Hey, sorry I didn't have notifications turned on for this feedstock. I'm surprised this happened, because the build did not change substantially between 10 & 11, at least on the conda side.

**Comment by h-vetinari** [comment]
Closing as fixed by #63; please let us know if something is still not working with `r-arrow=11.0.0=*_1`.

**Comment by darabos** [comment]
Awesome work, thanks a lot! 👍
